_config.yml

```
@@ -0,0 +1,46 @@
+title: OpenLoco
+name: OpenLoco Developers
+email: aaron@aaronweb.net
+description: An open source re-implementation of Chris Sawyer's Locomotion
+baseurl: "/OpenLoco-Pages"
+url: "https://aaronvangeffen.github.io"
```

Before merging this PR, change this to:
```suggestion
url: "https://openloco.io"
baseurl: "/"
```